docs/community/tutorials-contest/2022/avax-subnet-customization/README.md

```diff
@@ -0,0 +1,975 @@
+    Go back to previous step
+```
+
+Here you can set up a custom precompile to unlock some useful functions.
+
+##### Native Minting
+
+This precompile allows admins to permit designated contracts to mint the native token on your Subnet.
+We will discuss this in more detail in the [`contractNativeMinterConfig`](#contractNativeMinterConfig) section.
+
+##### Configure Contract Deployment Whitelist
+
+This precompile restricts who has the ability to deploy contracts on your Subnet.
+We will discuss this in more detail in the [`contractDeployerAllowListConfig`](#contractdeployerallowlistconfig) section.
+
+##### Configure Transaction Allow List
+
+This precompile restricts who has the ability to make transactions on your Subnet.
+We will discuss this in more detail in the [`txAllowListConfig`](#txAllowListConfig) section.
+
+However, we are not gonna do any of these yet. Choose `No` and proceed.
+
+```bash
+> Successfully created genesis
+```
+
+Congrats! You just created a Subnet! 👏🎉
+## Deploying the Subnet
+
+This is the most fun part. We will deploy the Subnet that we just created, and it's so easy to do, thanks to Avalanche CLI!
+
+```bash
+$ avalanche Subnet deploy testSubnet
+
+> ? Choose a network to deploy on:
+  ▸ Local Network
+    Fuji
+    Mainnet
+```
+
+We will deploy the Subnet on a local network so choose `Local Network` here.
+
+This process will install Avalanchego (if not already installed) and start the network. This will take a couple of minutes so, this is a good time to refresh your coffee ☕
+
+After this process is complete, you will see the Metamask connection details in the terminal.
+
+```bash
+Metamask connection details (any node URL from above works):
+RPC URL:          http://127.0.0.1:34249/ext/bc/8MiXurGFiuUfx3JzT4jXHVavgxDaBM4HSWotsGoeQMqRzcWWF/rpc
+Funded address:   0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC with 1000000 (10^18) - private key: 56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027
+Network name:     testSubnet
+Chain ID:         1234
+Currency Symbol:  TEST
+```
+
+You can use this configuration to add a new network to your metamask. Then you can import the account to which the airdrop funds were sent to into your metamask by importing the private key.
+
+![Metamask network config](.github/images/metamask.png)
+
+Notice that you will see the same private key if you've chosen `Airdrop 1 million tokens to the default address (do not use in production)` in the [airdrop step](#how-would-you-like-to-distribute-funds).
+Thus, the private key is exposed and anyone can access the account. For that reason, it's not a good idea to use this in production.
+
+After importing the account on metamask, let's change the network to our Subnet.
+
+![Metamask account page](.github/images/metamask2.png)
+
+How awesome is that? `TEST` is the native token of our Subnet and it's airdropped to our address. You can now deploy smart contracts and interact with the network.
+
+# Understanding the Genesis File
```

Comment:
Per [Markdownlint](https://github.com/DavidAnson/markdownlint) You shouldn't have multiple `<h1>`s in the same document.

> MD025/single-title/single-h1: Multiple top-level headings in the same document

[More info: MD025](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document)